lib/kubernetes-deploy/kubernetes_resource/deployment.rb

```diff
@@ -4,68 +4,54 @@ class Deployment < KubernetesResource
     TIMEOUT = 5.minutes
 
     def sync
-      json_data, _err, st = kubectl.run("get", type, @name, "--output=json")
+      raw_json, _err, st = kubectl.run("get", type, @name, "--output=json")
       @found = st.success?
-      @rollout_data = {}
+      @rollout_data = { "replicas" => 0 }
       @status = nil
-      @representative_pod = nil
-      @pods = []
+      @latest_rs = nil
 
       if @found
```

Comment:
I know is not a change here, but why is there no defined behaviour for if not found?